### PR TITLE
Fix fonts in Windows for CKJ langs

### PR DESCRIPTION
**Pull request by ylatuya**

### CODE REVIEW

**Comment by vguzmanp** [comment]
You are missing the pango.aliases change in roboto recipe too

**Comment by ylatuya** [comment]
This recipe is in another repo, don't ask me why...

File: recipes/pango.recipe [hunk 1]
```diff
@@ -30,3 +34,9 @@ class Recipe(recipe.Recipe):
         shell.replace(os.path.join(self.build_dir, 'Makefile'),
            {'SUBDIRS = pango modules pango-view examples docs tools tests build':
             'SUBDIRS = pango modules build'})
+
+    def post_install(self):
+        if self.config.target_platform == Platform.WINDOWS:
+            shutil.copy(self.relative_path("pango/pango.aliases"),
+                    os.path.join(self.config.prefix, "etc/pango/"))
```
[the remark this answers is not in the document]
Comment (ylatuya) [reply recipes/pango.recipe]: For CKJ langs the first option was always to use the system fonts, but Dani did the first try and couldn't make it work, that's why we tried to fallback to installing Noto. But it's perfectly possible to have system fonts working, so we go to the original plan

[the remark this answers is not in the document]
Comment (vguzmanp) [reply recipes/pango.recipe]: 👌 👍 when the roboto PR is up too

**Pull request merged**